vufs/src/main/java/com/stnetix/ariaddna/vufs/BusinessObjects/Metafile.java

```diff
@@ -0,0 +1,101 @@
+package com.stnetix.ariaddna.vufs.BusinessObjects;
+
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Metafile it's one of many vertex acyclic graph and on local filesystem metafile equals local files and directory.
+ */
+public class Metafile {
+
+    private String mVersion;
+    private String mFileUuid;
+    private String mParentFileUuid;
+    private Set<String> mChildFileUuidSet;
+    private List<String> mBlockUuidList;
+    private Map<String, String> mProperties;
+    private AllocationStrategy mAllocationStrategy;
+    private Map<String, String> mBlockAllocateMap;
+
+    public Metafile(String mVersion, String mFileUuid, String mParentFileUuid) {
+        mChildFileUuidSet = new HashSet<>();
```

Comment:
Because VUFS struct transmit to many services and it's not immutable structure, I think we should use treadsafe collection.